src/CoreBundle/DataAccess/DataAccessCmsMasterPagedefDatabase.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\DataAccess;
+
+use ChameleonSystem\CoreBundle\DataModel\CmsMasterPagdef;
+use ChameleonSystem\CoreBundle\Util\InputFilterUtilInterface;
+use TCMSPagedef;
+use TdbCmsMasterPagedef;
+use TGlobal;
+
+class DataAccessCmsMasterPagedefDatabase implements DataAccessCmsMasterPagedefInterface
```

Comment:
file should be used directly - for backend pagedefs. Database is fine - it just is also passed a fallback which happens to be the file version. But I renamed the injected data access into fallbackLoader.

Also note, that the code was extracted from the controller. Seeing as we plan on adding the change in a patch level version, I would like to avoid refactoring the code itself.